app/src/main/java/org/jellyfin/mobile/player/source/JellyfinMediaSource.kt

```diff
@@ -108,4 +109,8 @@ class JellyfinMediaSource(
             )
         } else null
     }
+
+    fun updateStartTimeTicks(startTime: Long) {
+        startTimeTicks = startTime * Constants.TICKS_PER_MILLISECOND
+    }
```

Comment:
The name might imply you need to pass ticks – I'd just call this `updateStartTime`, *or* add a custom setter to `startTimeMs` that delegates to `startTimeTicks`.